Solr/5/templates/schema.ss

```diff
@@ -32,9 +32,9 @@
  PERFORMANCE NOTE: this schema includes many optional features and should not
  be used for benchmarking.  To improve performance one could
   - set stored="false" for all fields possible (esp large fields) when you
-    only need to search on the field but don't need to return the original
+    only need to search on the field but don"t need to return the original
```

Comment:
```suggestion
    only need to search on the field but don't need to return the original
```

Reply:
I think I've been a bit too trigger-happy with the find-and-replace :laughing: